contrib/thermald/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "thermald"
+pkgver = "2.5.4"
+pkgrel = 0
+archs = [
+    "x86_64",
+]
```

Comment:
```suggestion
archs = ["x86_64"]
```

don't need a list for 1 entry